sdcm/cluster_k8s/mini_k8s.py

```diff
@@ -519,7 +519,7 @@ def start_k8s_software(self) -> None:
             labels:
               {POOL_LABEL_NAME}: {self.SCYLLA_POOL_NAME}
         """
-        for _ in range(self.params.get("n_db_nodes")):
+        for _ in range(self.params.get("n_db_nodes") + self.params.get("n_loaders") or 0):
```

Comment:
It will fail with TypeError exception if the `n_loaders` option is `None` (not set).
Need to make it be more safe.

Reply:
I've removed this code , it was wrong anyhow, need to make a separate loop for the loader nodes